fa25-team-b/scripts/folium_map_generation.py

```
@@ -0,0 +1,1402 @@
+    ensure_dir(os.path.dirname(cache_path))
+    with open(cache_path, 'w') as f:
+        json.dump(ma_geojson, f)
+
+    return cache_path
+
+def create_pie_chart_marker_html(row, demo_cols_dict, property_id):
+    """Create HTML for pie chart marker with unique ID for highlighting. Returns (html, size)."""
+    demo_values = []
+    demo_colors = []
+
+    for col, (label, color) in demo_cols_dict.items():
```

Ruff (0.14.7)

Loop control variable `label` not used within loop body

Rename unused `label` to `_label`

(B007)